pages/search/search.jsx

```diff
@@ -48,6 +52,24 @@ class Search extends React.Component {
     this.props.history.push(location);
   }
 
+  handleHelpChange(event) {
+    let helpType = event.target.value;
+
+    ReactGA.event({
+      category: `Search`,
+      action: `Help filtered`,
+      label: helpType
+    });
+
+    this.setState(
```

Comment:
Let's add a helper method so both `handleHelpChange` and `helpInputChange` can utilize

something like:
```jsx
  setSearchCriteria(key, value) {
    let criteria = {
      keywordSearched: ``,
      helpType: ``
    };

    criteria[key] = value;

    this.setState(criteria, () => {
      this.updateBrowserHistory();
    });
  }
```

Reply:
what should the purpose be of the helper method?

Reply:
So instead of having 

(in `handleHelpChange`)
```jsx
    this.setState(
      {
        helpType: helpType, 
        keywordSearched: `` 
      }, () => {
        this.updateBrowserHistory();
      });
```

and (in `helpInputChange`)
```jsx
    this.setState(
      {
        keywordSearched: keywordsEntered,
        helpType: ``
      }, () => {
        this.updateBrowserHistory();
      });
```

1. add a new helper method like what I suggested in my previous comment
2. replace the above code snippet with a function call to `this.setSearchCriteria`

e.g., ```this.setSearchCriteria(`helpType`, helpType)```